# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/GenerateCacheIRFiles.py
# Commit: 954c1651adb2
# Full Hash: 954c1651adb203c44392e127d6487aec3de53c97
# Author: Iain Ireland <iireland@mozilla.com>
# Date: 2020-07-07 09:47:47
# Regressor Bug: 1646378
# File Overlap Count: 1
# Description:
#   Bug 1646378: Add CallInlinedFunction r=jandem
#   
#   This is mostly the same as CallScriptedFunction. The key differences are:
#   
#   1. We have an extra argument, the ICScript, which must be stored in the JSContext so the Baseline prologue can find it.
# ==============================================================================

diff -r 01851025fa25 -r 954c1651adb2 js/src/jit/GenerateCacheIRFiles.py
--- a/js/src/jit/GenerateCacheIRFiles.py	Thu Jul 02 23:02:23 2020 +0000
+++ b/js/src/jit/GenerateCacheIRFiles.py	Mon Jul 06 19:05:21 2020 +0000
@@ -314,6 +314,7 @@
 
     return code
 
+
 def gen_clone_method(name, args):
     """Generates code for cloning a single opcode."""
 
@@ -364,6 +365,7 @@
 
     return code
 
+
 # Length in bytes for each argument type, either an integer or a C++ expression.
 # This is used to generate the CacheIROpArgLengths array. CacheIRWriter asserts
 # the number of bytes written matches the value in that array.